common/views/pages/_app.js

```diff
@@ -108,6 +108,31 @@ function openingHoursToOpeningHoursSpecification(openingHours: OpeningHours) {
   };
 }
 
+function makeSurePageIsTallEnough() {
+  const pageHeightCache = [];
+  const html = document.querySelector('html');
+
+  Router.events.on('routeChangeStart', () => {
+    document &&
```

Comment:
Why do we need to be so strict?

Reply:
Flow wasn't happy without it; is there a more sensible alternative?

Reply:
Na, seems OK then...